web/README.md

```diff
@@ -223,3 +227,111 @@ Use extreme caution when push pulling components between spaces as you can overw
    - Navigate to the web directory
    - Execute the push / pull command: npm run danger-storyblok-push-pull
    - Follow the prompts in the terminal, double checking the push / pull direction, space ids and components to push/pull.
+
+
+## Local setup between Next-Web and Publications
+
+### Publications setup
+Follow these steps to set up the Publications service locally and enable communication with local Next-Web.
+
+1. **Follow the Publications README:**
+   - Refer to the [Publications](https://github.com/nice-digital/publications) repository README and complete all the steps for local setup.
+2. **Modify `web.config` for local development:**
+   - In [web.config](https://github.com/nice-digital/publications/blob/master/src/Publications/web.config#L116), update the value of `UseSecureFeeds` to `false`
+   - Modify the value of `Non-Idam-EndpointKey-NextWeb` in `web.config` with the corresponding key from [Octodeploy](https://deploy-aws.nice.org.uk/app#/Spaces-1/projects/publications/variables).
+3. **Build and run the application:**
+   - Once the configuration is updated, build and run the Publications service as per the repository instructions.
+
+### Next-Web setup
+1. **Follow setup instructions:**
+   - Start by following the steps outlined in the [Set up](#rocket-set-up) section 
+2. **Create a `local-development.yml` file:**
+   - In the web/config directory of `Next-Web`, create a `local-development.yml` file and add the following configuration:
+
+```yaml
+public:
+	search:
+		baseURL: https://alpha-search-api.nice.org.uk/api
+	jotForm:
+		baseURL: https://nice.jotform.com
+server:
+	feeds:
+		publications:
+			# origin: https://alpha-publications.nice.org.uk
+			# apiKey: <Non-Idam-EndpointKey-NextWeb>
+			# origin: https://beta.publications.nice.org.uk
+			# apiKey: <Non-Idam-EndpointKey-NextWeb>
+			# origin: https://test-publications.nice.org.uk
+			# apiKey: <Non-Idam-EndpointKey-NextWeb>
+			# origin: https://live-publications.nice.org.uk
+			# apiKey: <Non-Idam-EndpointKey-NextWeb>
+			origin: https://local-publications.nice.org.uk
+			apiKey: <Non-Idam-EndpointKey-NextWeb>
+		inDev:
+			# origin: http://local-indev.nice.org.uk
+			# apiKey: <alpha-apikey>
+			origin: https://alpha-indev.nice.org.uk
+			apiKey: <alpha-apikey>
+			# origin: https://beta-indev.nice.org.uk
+			# apiKey: <beta-apikey>
+			# origin: https://test-indev.nice.org.uk
+			# apiKey: <test-apikey>
+			# origin: https://indev.nice.org.uk
+			# apiKey: <live-apikey>
+		jotForm:
+			apiKey: <jotform-apikey>
```

Comment:
We normally just write 'SECRET' for keys.

Reply:
Done